pkgpanda/http/config.py

```diff
@@ -0,0 +1,12 @@
+DCOS_ROOT = '/opt/mesosphere/'
```

Comment:
any way we can share getting / choosing these defaults sanely with the cli code?